test/Feature/OptionsSpec.hs

```diff
@@ -16,11 +16,40 @@ spec = describe "Allow header" $ do
     r <- request methodOptions "/items" [] ""
     liftIO $
       simpleHeaders r `shouldSatisfy`
-        matchHeader "Allow" "GET,POST,PATCH,DELETE"
+        matchHeader "Allow" "GET,POST,PUT,PATCH,DELETE,HEAD,OPTIONS"
 
   it "includes read verbs for read-only table" $ do
     r <- request methodOptions "/has_count_column" [] ""
     liftIO $
       simpleHeaders r `shouldSatisfy`
-        matchHeader "Allow" "GET"
+        matchHeader "Allow" "GET,HEAD,OPTIONS"
 
+  it "includes read verbs for non auto updatable views" $ do
+    r <- request methodOptions "/projects_view_without_triggers" [] ""
+    liftIO $
+      simpleHeaders r `shouldSatisfy`
+        matchHeader "Allow" "GET,HEAD,OPTIONS"
+
+  it "includes read/write verbs for insertable, updatable and deletable views" $ do
+    r <- request methodOptions "/projects_view_with_all_triggers" [] ""
+    liftIO $
+      simpleHeaders r `shouldSatisfy`
+        matchHeader "Allow" "GET,POST,PUT,PATCH,DELETE,HEAD,OPTIONS"
```

Comment:
And we need a test-case for a view that is updateable, but does not have a PK (just don't select the PK columns) - this should then return `POST` and `PATCH`, but not `PUT`.

Reply:
I organized the tests to make them more readable, and added the ones that were missing for views which are insertable and updatable but without pk.